### PR TITLE
added build gradle template file

### PR DESCRIPTION
**Pull request by davydotcom**

### CODE REVIEW

**Comment by craigburke** [comment]
I'll need a couple days to review this.

**Comment by davydotcom** [comment]
## Np. Ill work on getting docs for it

David Estes

On October 10, 2014 at 5:37:01 PM, Craig Burke (notifications@github.com) wrote:

I'll need a couple days to review this.

**Comment by craigburke** [comment]
@davydotcom after further reflection I think it makes more sense for this project to use a Microsoft batch file for the build. While I appreciate the contribution, I unfortunately can't use it. 

Since most of the contributors do most of their development on Windows XP machines I think rejecting this is better for the long term success of this project. Thanks again for the pull request!

**Comment by craigburke** [comment]
Wait a second... I just realized we can use Gradle to download a Windows VM for those stupid nerds who are afraid of GUIs and wizards and then run the build batch script within the VM. I'm reopening this ticket, but lets be clear Gradle is a means to an end only.

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    jcenter()
+	mavenCentral()
+ //   maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+}
+
+dependencies {
+	compile 'org.codehaus.groovy:groovy-all:2.2.0'
+	compile 'org.apache.ant:ant:1.7.0'
+	compile 'log4j:log4j:1.2.16'
+    compile "org.spockframework:spock-core:0.7-groovy-2.0"
```
Comment (craigburke) [inline build.gradle:27]: Please remove Star Trek references (Spock).

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,65 @@
+apply plugin: 'groovy'
+apply plugin: 'idea'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+
+version = '0.1.0'
```
Comment (craigburke) [inline build.gradle:6]: We should probably start at version 12 or 13 so people know the project is mature.

**Comment by craigburke** [comment]
I'm rewriting this whole thing in Kotlin so I'm closing this PR.

**Pull request closed without merging**